app/src/main/java/de/danoeh/antennapod/adapter/actionbutton/DownloadActionButton.java

```diff
@@ -17,6 +17,8 @@
 
 public class DownloadActionButton extends ItemActionButton {
 
+    private static boolean BYPASS_CELLULAR_NETWORK_WARNING = false;
```

Comment:
I would make this lower case. We use upper case only for constants elsewhere

Reply:
Done